Task 6/Task6.3/POINTING BLOCK.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset='utf-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <title>Task 8.1</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <div class="block">
+  <div class="triangl-border"></div>
+    <div class="triangl"></div>
+     
+    Дистанции огромного размера
+  </div>
```

Comment:
triangle is achievable with pseudo-elements before and after - they are connected to their element and this way you cannot forget to add these triangles in a correct order.